assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceTiles.js

```diff
@@ -60,9 +60,12 @@ export default function AudienceTiles( { Widget } ) {
 	const audiencesDimensionFilter = {
 		audienceResourceName: configuredAudiences,
 	};
-	const audiences = useSelect( ( select ) =>
-		select( MODULES_ANALYTICS_4 ).getAudiences()
-	);
+	const audiences = useSelect( ( select ) => {
+		const audienceData =
+			select( MODULES_ANALYTICS_4 ).getAvailableAudiences();
+
+		return Object.values( audienceData );
```

Comment:
This call to `Object.value()` should no longer be needed:

```suggestion
		return select( MODULES_ANALYTICS_4 ).getAvailableAudiences();
```